app/services/fuzzySearchService.js

```diff
@@ -101,10 +101,15 @@ const getSearchSuggestions = async (searchKey) => {
   if (!cachedGroupedPaymentData) {
     const paymentData = await getAllPaymentData()
     cachedGroupedPaymentData = await groupByPayee(paymentData)
+    // eslint-disable-next-line camelcase
+    cachedGroupedPaymentData = cachedGroupedPaymentData.map(({ scheme, total_amount, ...rest }) => rest)
   }
   const fuse = new Fuse(cachedGroupedPaymentData, fuseSearchOptions)
   const searchResult = fuse.search(searchKey).map(row => row.item)
-  return searchResult.slice(0, config.search.suggestionResultsLimit)
+  return {
```

Comment:
since this has been changed to an object from an array, you need to change the reference to records.length in searchSuggestions.js as length doesn't exist on an object type and the endpoint returns 404 error even when there is data.

Reply:
good catch, updated it now.